pkg/cluster/spec/util.go

```diff
@@ -107,3 +109,27 @@ func ClusterMetadata(clusterName string) (*ClusterMeta, error) {
 
 	return &cm, nil
 }
+
+// LoadClientCert read and load the client cert key pair and CA cert
+func LoadClientCert(dir string) (*tls.Config, error) {
+	return transport.TLSInfo{
+		TrustedCAFile: filepath.Join(dir, TLSCACert),
+		CertFile:      filepath.Join(dir, TLSClientCert),
+		KeyFile:       filepath.Join(dir, TLSClientKey),
+	}.ClientConfig()
+}
+
+// statusByURL queries current status of the instance by http status api.
+func statusByURL(url string, tlsCfg *tls.Config) string {
+	client := utils.NewHTTPClient(statusQueryTimeout, tlsCfg)
+
+	// body doesn't have any status section needed
+	body, err := client.Get(url)
+	if err != nil {
+		return "Down"
```

Comment:
I think we can add the error msg for more clarity？

Reply:
It has used to be, and been changed to this way for better understanding for users, having too many states might lead to confusion.